### PR TITLE
Added the option to create rules when you create a firewall

### PR DESCRIPTION
**Pull request by alejandrojnm**
Signed-off-by: Alejandro J. Nuñez Madrazo <alejandrojnm@gmail.com>

### CODE REVIEW

**Review by vishalanarase — APPROVED** [review]

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -4,23 +4,20 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.16.x, 1.17.x, 1.18.x]
```
Comment (RealHarshThakur) [inline .github/workflows/lint.yml:7]: Why are we dropping 1.16?

**Comment by dirien** [comment]
Hi @alejandrojnm, sorry to comment here but this feature broke an app of mine! I guess release v0.3.5 should have been marked with a breaking change?

see https://github.com/dirien/minectl/runs/7918141321?check_suite_focus=true#step:5:9

**Comment by alejandrojnm** [comment]
That is odd because I saw you have set it the version 0.3.3 in your go mod it should be ok, but is failing

**Comment by dirien** [comment]
@alejandrojnm np, just wanted to ask!

**Pull request merged**